etcd/datadog_checks/etcd/data/conf.yaml.example

```diff
@@ -17,128 +20,147 @@ init_config:
     #   http: http://<PROXY_SERVER_FOR_HTTP>:<PORT>
     #   https: https://<PROXY_SERVER_FOR_HTTPS>:<PORT>
     #   no_proxy:
-    #     - <HOSTNAME_1>
-    #     - <HOSTNAME_2>
+    #   - <HOSTNAME_1>
+    #   - <HOSTNAME_2>
+
+    ## @param skip_proxy - boolean - optional - default: false
+    ## If set to `true`, this makes the check bypass any proxy
+    ## settings enabled and attempt to reach services directly.
+    #
+    # skip_proxy: false
+
+    ## @param timeout - number - optional - default: 10
+    ## The timeout for connecting to services.
+    #
+    # timeout: 10
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
 
+## Every instance is scheduled independent of the others.
+#
 instances:
 
-  -  
+  -
     ## @param use_preview - boolean - optional - default: true
-    ## Whether or not to preview the new version of the check supporting only ETCD 3+
+    ## Whether or not the preview to new version of the check supporting only ETCD 3+
```

Comment:
I'm still not sure what this is saying? Whether or not to _use_ the preview of the new version? or Whether or not to preview the new version?

Reply:
To use the new version

Reply:
In that case can you change it to `Whether or not to use the preview to new version of the check supporting only ETCD 3+`